test_scripts/Polices/Policies_Security/015_P_ATF_PolicyTable_RAI_with_NULL_policies_RPCs_DISALLOWED.lua

```diff
@@ -0,0 +1,111 @@
+----- [GeneralResultCode] DISALLOWED. A request comes with appID which has "null" permissions in Policy Table
+----- In case PolicyTable has "<appID>": "null" in the Local PolicyTable for the specified application with appID, 
+----- PoliciesManager must return DISALLOWED resultCode and success:"false" to any RPC requested by such <appID> app.
+
```

Comment:
Please also add 1. Used preconditions (if applicable)
2. Performed steps 
3. Expected result 

Please check https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/27 for more details